From ac028114e60d129082f66a23aca01e76971e681b Mon Sep 17 00:00:00 2001
From: Charles Steinkuehler <cstein@newtek.com>
Date: Wed, 4 Oct 2023 19:33:48 -0500
Subject: [PATCH 1/3] Fix USB host controller on CM4

USB xhci compatible string needs to be modified to work with CM4 dtb

Signed-off-by: Charles Steinkuehler <cstein@newtek.com>
---
 drivers/usb/host/xhci-brcm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/xhci-brcm.c b/drivers/usb/host/xhci-brcm.c
index fe17924028..0c6938187b 100644
--- a/drivers/usb/host/xhci-brcm.c
+++ b/drivers/usb/host/xhci-brcm.c
@@ -82,7 +82,7 @@ static int xhci_brcm_deregister(struct udevice *dev)
 }
 
 static const struct udevice_id xhci_brcm_ids[] = {
-	{ .compatible = "brcm,generic-xhci" },
+	{ .compatible = "generic-xhci" },
 	{ }
 };
 
-- 
2.11.0

